spec/support/shared_examples/a_measure_unit_presentable.rb

```diff
@@ -0,0 +1,13 @@
+RSpec.shared_examples_for 'a measure unit presentable' do
+  describe '#presented_unit' do
+    it 'returns the properly presented unit' do
+      expect(evaluator.presented_unit).to eq(expected_presented_unit) if expected_presented_unit
+    end
+  end
+
+  describe '#applicable_unit' do
+    it 'returns the merged units from the ApplicableMeasureUnitMerger service' do
+      expect(evaluator.applicable_unit).to eq(expected_applicable_unit)
+    end
+  end
+end
```

Comment:
The evaluator is the subject :). I'm treating it as a parameter, actually. I could probably pull it explicitly into the shared example since the api is the same over and over for evaluators